src/Resources/symfony/ansible/group_vars/deploy_demo.yml

```diff
@@ -0,0 +1,19 @@
+
+---
+
+###########
+# Removed #
+###########
+
+manala_deploy_removed:
+  - web/app_dev.php
```

Comment:
For reference: https://github.com/Elao/symfony-standard/blob/master/ansible/group_vars/deploy_demo.yml